frontend/components/forms/fields/Dropdown/Dropdown.jsx

```diff
@@ -47,6 +48,7 @@ class Dropdown extends Component {
     placeholder: "Select one...", // if value undefined
     parseTarget: false,
     tooltip: "",
+    autoFocus: false,
```

Comment:
Same here, although some of the other prop names don't follow boolean naming conventions in this component so less confusing, though I still don't know what `multi` means.

Reply:
`multi` is 6 years old naming for multi select dropdowns which include selecting targets for queries, packs, and adding team members

Reply:
Yep, following the existing conventions is fine until we standardize.